test/test_autograd.py

```diff
@@ -2514,6 +2581,18 @@ class dont_convert(tuple):
      'distinct_postive_s', NO_ARGS, [skipIfNoLapack]),
     ('svd', lambda: random_fullrank_matrix_distinct_singular_value(S), NO_ARGS, '', NO_ARGS, [skipIfNoLapack]),
     ('gesv', (S, S), ((S, S),), '', NO_ARGS, [skipIfNoLapack]),
+    ('fft', lambda: torch.randn(5), (1, False, True, False, None, True, False), 'signal_dim1_r2c'),
+    ('fft', (5, 50), (1,), 'batch_signal_dim1_r2c'),
+    ('fft', (3, 3), (2,), 'signal_dim2_r2c'),
+    ('fft', (10, 20, 20), (2,), 'batch_signal_dim2_r2c'),
+    ('fft', (20, 20, 20), (2,), 'signal_dim3_r2c'),
+    ('fft', (10, 20, 20, 20), (2,), 'batch_signal_dim3_r2c'),
+    ('fft', (50, 2), (1, True), 'signal_dim1_c2c'),
+    ('fft', (5, 50, 2), (1, True), 'batch_signal_dim1_c2c'),
+    ('fft', (20, 20, 2), (2, True), 'signal_dim2_c2c'),
+    ('fft', (10, 20, 20, 2), (2, True), 'batch_signal_dim2_c2c'),
+    ('fft', (20, 20, 20, 2), (3, True), 'signal_dim3_c2c'),
+    ('fft', (10, 20, 20, 20, 2), (3, True), 'batch_signal_dim3_c2c'),
```

Comment:
Why aren’t we testing the other variants?

Reply:
I should delete these as well. I added a new test method `test_fft` above to test all four variants.